man/ocamlopt.1

```diff
@@ -753,6 +753,31 @@ Generate position-independent machine code.  This is the default.
 .B \-fno\-PIC
 Generate position-dependent machine code.
 
+.SH OPTIONS FOR FLAMBDA
+
+The Flambda code generator supports the following additional options:
```

Comment:
It migth be also useful to mention that those options are only available when the compiler has been configured with `-flambda`:

```
When the Flambda code generator has been enabled at configuration time, the following additional options are available:
```

Reply:
If I remember correctly, we made sure that passing flambda-specific options to a non-flambda compiler doesn't fail; the options are only ignored silently. So maybe `become meaningful` rather than `are available` ?

Reply:
Fine with me.